azure/src/wizard/StorageAccountCreateStep.ts

```diff
@@ -4,14 +4,18 @@
  *--------------------------------------------------------------------------------------------*/
 
 import type { SkuName, StorageManagementClient } from '@azure/arm-storage';
+import type { IAzureNamingRules } from '@microsoft/vscode-azext-utils';
+import { AzureWizardExecuteStep } from '@microsoft/vscode-azext-utils';
```

Comment:
I think it's fine to include both of these in the same `import` (not `import type`), as the utils package is designed to not need to be lazy-loaded.

Reply:
Ohh ok ok. Thanks!